posprocessing.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+def frames_borders(data, minf, maxf):
+    filter_data = []
+    for f in data:
+        if f > minf and f < maxf:
+            filter_data.append(f)
+    return filter_data
+
+
+def skip_unnecessary_frames(frames_with_steps, frames_start_second_double_support, frames_first_double_support,
```

Comment:
would it make sense to have funciton signature as 
```python
def skip_unnecessary_frames(frames_with_steps, frames _to_filter)
``` 
and call it 3 times for each of `frames_start_second_double_support`, `frames_first_double_support`, `frames_second_double_support` as a parameter?